sanic/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "19.12.0"
+__version__ = "19.12.2"
```

Comment:
Does the version need to be bumped for this?

Reply:
Nope. It was a mistake and was meant to be bumped on a previous commit.